packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -222,6 +224,13 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
               }}
             />
             <Grid>
+              <Row>
+                <ExtensionSlot extensionSlotName="breadcrumbs-slot">
+                  <div className={styles.breadcrumbsContainer}>
```

Comment:
Not sure - why do we need this container?

Reply:
(The breadcrumbs should look OK on any page when placed there; if not we should alter the default styling.)

Reply:
The breadcrumbs have a grey background, and I wanted the background to be transparent. 
Yes I also believe that the breadcrumbs should be transparent in nature, the grey background should not be default. 
Also about the padding, the default padding is 2rem, but I believe the padding should be `2rem 0` to be default, so that the user can align it on it's own, wherever needed.

Reply:
The padding we should keep as is (after all, this is the choice from the Carbon design language; which we inherit). The background, however, does not make much sense. We should override this in the `esm-core` repository and avoid wrapping the breadcrumbs here.

Reply:
Duly Noted, I'll push the required changes in the `esm-core`.
Thanks alot!